src/components/Wizard/Inputs/Integer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { Slider, SliderProps } from "baseui/slider";
+
+import { InputProps } from "./common";
+import useDebounce from "../../../hooks/useDebounce";
+import { FormControlBlock } from "./FormControlBlock";
+import { useFormField } from "../useForm";
+
+export function Input<FormValues>({
+  name,
+  min,
+  max,
+  step,
+  title,
+  help,
+}: IntegerInputProps<FormValues>) {
+  const { value, setValue, error } = useFormField<FormValues>(name);
+  const [currentValue, setCurrentValue] = React.useState<number>(
+    value as unknown as number || min || 0
```

Comment:
what is this `as unknown` voodoo you're doing here? I'm not familiar.

Reply:
This was to get around the potential mismatch in types (value could be anything, but IntegerInput would only take numbers).

I've updated that here https://github.com/raindrop-studios/creator-ui/pull/1/commits/f22e89b796585fa7f1ca9bae9ad6cacbeddce535 to pass the expected type into `useFormField` and handle it in there